contessa/consistency_checker.py

```diff
@@ -70,9 +70,16 @@ def run(
             context,
         )
 
-        quality_check_class = create_default_check_class(result_table)
-        self.right_conn.ensure_table(quality_check_class.__table__)
-        self.upsert(quality_check_class, result)
+        if result_table:
+            result_table = ResultTable(**result_table, model_cls=self.model_cls)
+            quality_check_class = create_default_check_class(result_table)
+            self.right_conn.ensure_table(quality_check_class.__table__)
+            self.upsert(quality_check_class, result)
+            return result
```

Comment:
I'm a bit worried that it returns different types. Is there a way to unify it, e.g. under CheckResult type?